k8s/jenkins/scripts/security.groovy

```diff
@@ -35,4 +106,63 @@ if (hudsonRealm.getAllUsers().size == 0){
 	Jenkins.instance.getInjector().getInstance(AdminWhitelistRule.class).setMasterKillSwitch(false)
 }
 
-System.setProperty("hudson.model.DirectoryBrowserSupport.CSP", "")
\ No newline at end of file
+enableOpenId = env['OPENID_ENABLE']
+if (enableOpenId.toLowerCase() == "true") {
+  String clientId = env['OPENID_CLIENTID']
```

Comment:
@MaksymBilenko  As I see, any job on Jenkins can access this env. variables with critical information. Am I right?

Reply:
All Jenkins jobs are running on slaves where we don't have this variables.